src/ConsoleArgument.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Console;
+
+use Attribute;
+use Tempest\Support\ArrayHelper;
+
+#[Attribute(Attribute::TARGET_METHOD | Attribute::TARGET_PROPERTY | Attribute::TARGET_PARAMETER)]
+final class ConsoleArgument
+{
+    /** @var string[] */
+    private array $helpMessages;
+
+    /**
+     * @param string|string[] $help
+     * @param string[] $aliases
+     */
+    public function __construct(
+        array|string $help = [],
+        public array $aliases = [],
+    ) {
+        $this->helpMessages = ArrayHelper::wrap($help);
+    }
+
+    /** @return string[] */
+    public function getHelpLines(): array
+    {
+        return $this->helpMessages;
+    }
+
+    /** @return string[] */
+    public function getAliases(): array
```

Comment:
Just an FYI, if property hooks make it into 8.4, I think it's a better design to not have getters anymore. So I'm fine dropping this getter and just relying on the public property